src/ui_nbgl.c

```diff
@@ -76,9 +76,20 @@ static void release_context(void)
 	}
 }
 
-static inline uint8_t get_element_line_count(const char* line)
+static inline uint16_t get_element_line_count(const char* line)
 {
-	return strlen(line) / MAX_TAG_CONTENT_CHAR_PER_LINE + 2;
```

Comment:
Probably allow you to drop some custom define like MAX_TAG_CONTENT_CHAR_PER_LINE?